docs/README.md

```diff
@@ -1,177 +1,69 @@
-| `popupChildrenContainer`            | The main container for children content inside the popup variant on the minicart | [Popup](/react/components/Popup.js)                     |
-| `footerShippingPriceContainer`      | The shipping price main container                                                | [MinicartFooter](/react/components/MinicartFooter.js)   |
-| `footerShippingPriceLabelContainer` | The shipping price label container                                               | [MinicartFooter](/react/components/MinicartFooter.js)   |
-| `footerShippingPriceLabelContainer` | The shipping price label container                                               | [MinicartFooter](/react/components/MinicartFooter.js)   |
-| `sidebarRightCaretContainer`        | The caret icon container in the sidebar variant of the minicart                  | [Sidebar](/react/components/Sidebar.js)                 |
-| `sidebarItemQuantityContainer`      | The item quantity container in the sidebar variant of the minicart               | [Sidebar](/react/components/Sidebar.js)                 |
-
-## Troubleshooting
-
-You can check if others are passing through similar issues [here](https://github.com/vtex-apps/minicart/issues). Also feel free to [open issues](https://github.com/vtex-apps/minicart/issues/new) or contribute with pull requests.
-
-## Contributing
-
-Check it out [how to contribute](https://github.com/vtex-apps/awesome-io#contributing) with this project.
-
-## Tests
-
-To execute our tests go to `react/` folder and run `yarn test`
-
-### Travis CI
-
-[![Build Status](https://api.travis-ci.org/vtex-apps/minicart.svg?branch=master)](https://travis-ci.org/vtex-apps/minicart)
+| Prop name              | Type      | Description                                                                                              | Default value |
+| ---------------------- | --------- | -------------------------------------------------------------------------------------------------------- | ------------- |
+| `variation`            | `Enum`    | Define Minicart variation. (values: `'popup'`, `'drawer'` or `'link'`)                                   | `'sidebar'`   |
+| `openOnHover`          | `Boolean` | Whether the popup minicart should open when the user hovers over it.                                     | false         |
+| `linkVariationUrl`     | `String`  | The link associated to the `'link'` variation of the Minicart.                                           | false         |
+| `drawerSlideDirection` | `Enum`    | Controls the slide direction for the `'sidebar'` variation. (values: `'rightToLeft'` or `'leftToRight'`) | `rightToLeft` |
+| `maxDrawerWidth`       | `Number`  | The maximum width in pixels for the open sidebar Minicart.                                               | `440`         |
+
+## Customization
```

Comment:
Victor, could you add the following sentence before the handles table?

`In order to apply CSS customizations in this and other blocks, follow the instructions given in the recipe on [Using CSS Handles for store customization](https://vtex.io/docs/recipes/style/using-css-handles-for-store-customization).`

Reply:
Sure!! Forgot about that!